### ISSUE TITLE
Cannot find module '../../.tmp/services-list'

### ISSUE THREAD

**Issue by acollierr17** [dbots-pkg/dbots.js#23]
After recently implementing the library into my bot and running it on production, I came across the error in the title. I also included the stack trace below. I ended up reverting the change and will be on standby for more information.

Here's some information on my setup:

- Discord Library + Version: Discord.js v12.2.0
- Container Image: Node LTS (`node:lts`) - v12.18.2
- Docker v19.03.4 build 9013bf583a
- Docker Compose v1.18.0 build 8dd22a9
- Machine OS: CentOS Linux 7 (Core)
- Machine Kernel: Linux 3.10.0-1062.4.1.el7.x86_64

If there's any further information I need to provide, please, let me know.

*Edit: Fix container image version*
```
Error: Cannot find module '../../.tmp/services-list',
- /usr/src/suggestions-bot/node_modules/dbots/src/Interface/ServiceBase.js,
Require stack:,
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10),
    at Module.load (internal/modules/cjs/loader.js:986:32),
    at Module._compile (internal/modules/cjs/loader.js:1138:30),
    at Object.<anonymous> (/usr/src/suggestions-bot/node_modules/dbots/src/Interface/ServiceBase.js:153:20),
    at require (internal/modules/cjs/helpers.js:72:18),
    at Module.require (internal/modules/cjs/loader.js:1026:19),
    at Function.Module._load (internal/modules/cjs/loader.js:842:27),
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15),
- /usr/src/suggestions-bot/bot.js,
- /usr/src/suggestions-bot/client/SuggestionsClient.js,
- /usr/src/suggestions-bot/node_modules/dbots/src/index.js,
```

**Comment by EndBug** [comment]
Hi, thanks for the issue.: I think this is happening because you set the `NODE_ENV` variable to `production`, which makes our code think that we're running webpack. I'll push a fix right now ;)

**Comment by EndBug** [comment]
Ok @acollierr17, the issue should be fixed now. Could you try testing the package by temporarily installing it using this repo?
```
npm i dbots-pkg/dbots.js#master
```

**Comment by acollierr17** [comment]
> Hi, thanks for the issue. I think this is happening because you set the `NODE_ENV` variable to `production`, which makes our code think that we're running webpack. I'll push a fix right now ;)

This is correct.



> Ok @acollierr17, the issue should be fixed now. Could you try testing the package by temporarily installing it using this repo?
> 
> ```
> npm i dbots-pkg/dbots.js#master
> ```

I uninstalled the previous package and installed the temporary package (`dbots-pkg/dbots.js#master`). After pushing the changes to production, I still came up with the same error.

**Comment by EndBug** [comment]
I need you to post the error again, since I've tried reproducing it gets fixed on my machine (while I see it using the npm version, as you reported). 

Please check the following:
- Run `npm uninstall dbots && npm install dbots-pkg/dbots.js#master`
- Check that your package file shows `github:dbots-pkg/dbots.js#master` in the dependencies
- Run your code from the start

If after those steps the error keeps showing up, please post it along with the whole stack trace (as you did before)

**Comment by EndBug** [comment]
Also, please log the values of `process.env.NODE_ENV` and `process.release.name` and add them to your comment, so that we can see if something is going wrong there.

**Comment by acollierr17** [comment]
Alright, for some reason, I was thinking it actually installed from the right location when I initially reinstalled dbots.js using `dbots-pkg/dbots.js#master`. I did it again and the issue is resolved.

**Comment by EndBug** [comment]
Alright, the patch has been published as version 5.0.4, you can now install dbots from your usual package manager :)
Thanks for your issue, I'll close it for now ;)
